hertzbeat-common/src/main/resources/msg.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+manager_import_task_progress=导入任务[%s]进度: %d%%
+manager_import_task_success=导入任务[%s]完成
+manager_import_task_fail=导入任务[%s]失败: %s
```

Comment:
👍👍 hi, how about use the webui i18n json instead of the backend i18n properties. Due the information displayed here is the front-end information, we can unify it in the front-end i18n. Our webapp i18n is more complete, and we will gradually use them in the future.